src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/boss_grand_champions.cpp

```diff
@@ -15,577 +16,1264 @@
+    SPELL_INTERCEPT                     = 67540,
+    SPELL_ROLLING_THROW                 = 67546,
 
     // Ambrose Boltspark && Eressea Dawnsinger || Mage
-    SPELL_FIREBALL                  = 66042,
-    SPELL_FIREBALL_H                = 68310,
-    SPELL_BLAST_WAVE                = 66044,
-    SPELL_BLAST_WAVE_H              = 68312,
-    SPELL_HASTE                     = 66045,
-    SPELL_POLYMORPH                 = 66043,
-    SPELL_POLYMORPH_H               = 68311,
+    SPELL_FIREBALL                      = 66042,
+    SPELL_BLAST_WAVE                    = 66044,
+    SPELL_HASTE                         = 66045,
+    SPELL_POLYMORPH                     = 66043,
 
     // Colosos && Runok Wildmane || Shaman
-    SPELL_CHAIN_LIGHTNING           = 67529,
-    SPELL_CHAIN_LIGHTNING_H         = 68319,
-    SPELL_EARTH_SHIELD              = 67530,
-    SPELL_HEALING_WAVE              = 67528,
-    SPELL_HEALING_WAVE_H            = 68318,
-    SPELL_HEX_OF_MENDING            = 67534,
+    SPELL_CHAIN_LIGHTNING               = 67529,
+    SPELL_EARTH_SHIELD                  = 67530,
+    SPELL_HEALING_WAVE                  = 67528,
+    SPELL_HEX_OF_MENDING                = 67534,
 
     // Jaelyne Evensong && Zul'tore || Hunter
-    SPELL_DISENGAGE                 = 68340, //not implemented in the AI yet...
-    SPELL_LIGHTNING_ARROWS          = 66083,
-    SPELL_MULTI_SHOT                = 66081,
-    SPELL_SHOOT                     = 65868,
-    SPELL_SHOOT_H                   = 67988,
-
-    // Lana Stouthammer Evensong && Deathstalker Visceri || Rouge
-    SPELL_EVISCERATE                = 67709,
-    SPELL_EVISCERATE_H              = 68317,
-    SPELL_FAN_OF_KNIVES             = 67706,
-    SPELL_POISON_BOTTLE             = 67701
+    SPELL_DISENGAGE                     = 68339, // 68340 is correct disengage but spell needs to be scripted
+    SPELL_LIGHTNING_ARROWS              = 66083,
+    SPELL_LIGHTNING_ARROWS_AURA         = 66085,
+    SPELL_MULTI_SHOT                    = 66081,
+    SPELL_SHOOT                         = 66079,
+
+    // Lana Stouthammer && Deathstalker Visceri || Rouge
+    SPELL_EVISCERATE                    = 67709,
+    SPELL_FAN_OF_KNIVES                 = 67706,
+    SPELL_POISON_BOTTLE                 = 67701,
+    SPELL_DEADLY_POISON                 = 67711
 };
 
-enum Seats
+enum Texts
 {
-    SEAT_ID_0                       = 0
+    EMOTE_TRAMPLE                       = 0
 };
 
-/*
-struct Point
+enum PointMovement
 {
-    float x, y, z;
+    POINT_REMOUNT                       = 0,
+    POINT_DESPAWN
 };
 
-const Point MovementPoint[] =
-{
-  {746.84f, 623.15f, 411.41f},
-  {747.96f, 620.29f, 411.09f},
-  {750.23f, 618.35f, 411.09f}
-};
-*/
-void AggroAllPlayers(Creature* temp)
+void EnterAggressiveMode(Creature* temp)
 {
-    Map::PlayerList const& PlList = temp->GetMap()->GetPlayers();
+    temp->RemoveFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NON_ATTACKABLE | UNIT_FLAG_IMMUNE_TO_PC);
```

Comment:
this was commented at https://github.com/TrinityCore/TrinityCore/pull/21427#discussion_r169184363 but it wasn't resolved